dir.traversal.targets

```diff
@@ -54,56 +54,20 @@
     <Error Condition="'$(MSBuildLastTaskResult)'=='false'" />
   </Target>
 
-  <!-- FilterProjectsToTest will filter the project list according to the FilterToTestTFM value -->
-  <Target Name="FilterProjectsToTest"
-          BeforeTargets="TestAllProjects"
-          Condition="$(MSBuildProjectName.EndsWith('tests'))">
-
-    <ItemGroup>
-      <Project>
-        <!-- default to DefaultTestTFM defined in root dir.props if TestTFMs aren't set on the project -->
-        <TestTFMs Condition="'%(Project.TestTFMs)'==''">$(DefaultTestTFM)</TestTFMs>
-      </Project>
-      <Project>
-        <!-- make sure every defined TestTFM value are surrounded by semicolons for easier search, e.g. ";net46;"" -->
-        <TestTFMs>;%(Project.TestTFMs);</TestTFMs>
-      </Project>
-
-      <!-- Include all projects which have TestTFM from the supported set -->
-      <ProjectsToTest Include="@(Project)" Condition="'%(Project.Extension)'=='.csproj' And $([System.String]::new('%(Project.Identity)').Contains('.Tests'))">
```

Comment:
Previously we were only testing projects that ended with "tests" but I don't think it makes sense to filter this in multiple places.  Let just let the builds file define the set of projects.  I fixed the one case that wasn't working, RemoteExecutorConsoleApp: it was failing because we hardcoded `.dll` as the test extension.  I now use TargetFileName.  This will run xunit against the assembly which is a noop since it contains no tests.  If we want to bring back this filtering let me know.  /cc @weshaggard @mellinoe @joperezr